chapters/dae.tex

```diff
@@ -102,10 +102,12 @@ \chapter{Modelica DAE Representation}\label{modelica-dae-representation}
 \end{example}
 
 The generated set of equations is used for simulation and other analysis activities.
-Simulation means that an initial value problem is solved, i.e., initial values have to be provided for the states $x$, \cref{initialization-initial-equation-and-initial-algorithm}.
+Simulation proceeds as follows.
+First, initial values are found for the states $x$, \cref{initialization-initial-equation-and-initial-algorithm}.
+Given those initial values the equations are simulated forward in time; this is called the transient analysis.
```

Comment:
Make it sound less like this is the definition (which would have required the use of `\firstuse`):
```suggestion
Given those initial values the equations are simulated forward in time; this is the transient analysis.
```